psydac/linalg/tests/test_stencil_vector.py

```diff
@@ -198,8 +198,8 @@ def test_stencil_2d_array_to_stencil( n1, n2, p1, p2, P1, P2 ):
 #===============================================================================
 # PARALLEL TESTS
 #===============================================================================
-@pytest.mark.parametrize( 'n1', [8,23] )
-@pytest.mark.parametrize( 'n2', [8,25] )
+@pytest.mark.parametrize( 'n1', [12,22] )
+@pytest.mark.parametrize( 'n2', [12,24] )
```

Comment:
Some number of cells don't work because of the requirement that we have, where we get [this](https://github.com/pyccel/psydac/blob/devel-mp-ddm/psydac/ddm/partition.py#L125) error